apiary-metastore-events/apiary-metastore-consumers/privileges-grantor/privileges-grantor-lambda/src/test/java/com/expediagroup/apiary/extensions/events/metastore/consumer/privilegesgrantor/lambda/PrivilegesGrantorTest.java

```diff
@@ -1,124 +0,0 @@
-/**
```

Comment:
This file was in the wrong place and already existed in the right place, so I've deleted it.